### PR TITLE
Add missing custom field in draft, impl and builder of orderImport. Close #792

### LINKED ISSUE #792: Add missing fields to OrderImport
like custom fields

http://dev.sphere.io/http-api-projects-orders-import.html

### PR DESCRIPTION
**Pull request by ccsalazarr-commercetools**
Add missing custom field in draft, impl and builder of orderImport. Close #792

### CODE REVIEW

**Comment by ccsalazarr-commercetools** [comment]
@katmatt could you review it? please

File: commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static io.sphere.sdk.test.SphereTestUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OrderImportDraftBuilderTest {
+
+    @Test
+    public void createsFromOrderImportDraft() throws Exception {
```
Comment (katmatt) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java]: I would name this test like `constructWithAllValuesSet`.

File: commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        final LocalizedEnumValue yellow = LocalizedEnumValue.of("yellow", LocalizedString.of(ENGLISH, "yellow").plus(GERMAN, "gelb"));
```
Comment (katmatt) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java]: We should try to use our fixture classes here.

File: commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+import org.junit.Test;
+
+import javax.money.MonetaryAmount;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static io.sphere.sdk.test.SphereTestUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OrderImportDraftBuilderTest {
```
Comment (katmatt) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java]: I like that you created a unit test for this builder 👍

File: commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java (hunk 2 above)
Comment (ccsalazarr-commercetools) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java]: @katmatt I think that if we use for example `ProductFixtures`, we would have to transform the test into an IntegrationTest, as the already existing methods in the fixture are using the backend to create the products. or what fixtures do you have in mind?

Comment (katmatt) [inline commercetools-models/src/test/java/io/sphere/sdk/orders/OrderImportDraftBuilderTest.java]: You're right and I was wrong. And then I would suggest to keep your code.

**Pull request merged**